packages/components/src/mobile/inserter-button/index.native.js

```diff
@@ -10,14 +10,13 @@ import { Component } from '@wordpress/element';
 import { Icon } from '@wordpress/components';
 import { withPreferredColorScheme } from '@wordpress/compose';
 import { __, sprintf } from '@wordpress/i18n';
-import { sparkles } from '@wordpress/icons';
 import { BlockIcon } from '@wordpress/block-editor';
 
 /**
  * Internal dependencies
  */
 import styles from './style.scss';
-
+import { sparkles } from './sparkles';
```

Comment:
I noticed that the icon is exported as `default`, so it would be great if we could update this with:
```suggestion
import sparkles from './sparkles';
```